rbm/model.py

```
@@ -0,0 +1,163 @@
+from tinygrad import Tensor, nn
```

Ruff (0.11.9)

`tinygrad.nn` imported but unused

Remove unused import: `tinygrad.nn`

(F401)